scripts/visualize_perspective_transforms.py

```
@@ -0,0 +1,365 @@
+        # Save the visualization
+        output_path = output_dir / f"{image_id}_comparison.jpg"
+        pil_image.save(output_path, quality=90)
+        logger.info(f"Saved: {output_path}")
+
+        # Clean up downloaded image
+        if image_path.exists():
+            image_path.unlink()
+
+        return str(output_path)
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)